rq_function.py

```diff
@@ -0,0 +1,21 @@
+import requests
+import os
+
+def add(a,b):
+    print(a+b)
+
+def count_lines(path):
```

Comment:
This is excellent, having a separation between the functions doing the work and routing and queuing logic will make this much easier to test and think about. You could also look at separating the queuing and flask stuff.

What might be useful is looking at pytest or another testing library for writing unit tests of these functions. And of your endpoints in app.py.